package.json

```diff
@@ -52,6 +51,7 @@
     "postcss-reporter": "^1.3.0",
     "psi": "^2.0.2",
     "react-addons-test-utils": "^0.14.6",
+    "react-router-redux": "^2.1.0",
```

Comment:
Is there a reason for adding react-router-redux on the `devDependencies` list instead of `dependencies`?